### DISCUSSION TITLE
Rime插件打开使用者资料夹

### DISCUSSION THREAD

**Discussion by qwert-f** [fcitx5-android/fcitx5-android#468] [Q&A | 问答] [answered]
我本机安装了wallpaper engine每次点那个插件的使用者资料夹都会打开wallpaper engine或者报错，这是怎么回事呀

**Comment by rocka** [comment, marked as answer]
你可以尝试更新到 CI 最新版本： https://jenkins.fcitx-im.org/job/android/job/fcitx5-android/

应该已经修复此问题

**Reply by qwert-f** [reply]
谢谢，问题已修复
